src/main/docker/Dockerfile

```
@@ -15,8 +15,11 @@
 #
 
 ARG DOCKER_HUB_PUBLIC=docker.io
-FROM ${DOCKER_HUB_PUBLIC}/digitallyseamless/nodejs-bower-grunt:5
+FROM ${DOCKER_HUB_PUBLIC}/cafapi/opensuse-nodejs10:latest
```

We typically use the 1-part version number when referencing other images (so that we get updates but not breaking changes):

```suggestion
FROM ${DOCKER_HUB_PUBLIC}/cafapi/opensuse-nodejs10:2
```